src/components/VotingPower/index.tsx

```diff
@@ -4,86 +4,110 @@ import 'tippy.js/dist/tippy.css'
+          </span>
+          <span>{BNtoDecimal(currentVotingPower, 0, 2)}</span>
+        </S.YourVotingPower>
+      )}
+
+      {yourVotingPowerInProposal && (
+        <S.YourVotingPower>
+          <span>
+            Snapshot voting power
             <Tippy content="Amount of voting power snapshoted from your address to vote on this proposal. This voting power is relative to the block in which the proposal was published.">
               <S.Tooltip tabIndex={0}>
-                <Image src={tooltip} alt="Explanation" width={14} height={14} />
+                <Image
+                  src={tooltip}
+                  alt="tooltip about your voting power at the moment this proposal was created"
+                  width={14}
+                  height={14}
+                />
               </S.Tooltip>
             </Tippy>
-          )}
-        </span>
-        <span>
-          {yourVotingPowerInProposal === undefined
-            ? BNtoDecimal(Big(data?.user?.votingPower ?? Big(0)), 0, 2)
-            : BNtoDecimal(
-                yourVotingPowerInProposal.div(Big(10).pow(18)),
-                18,
-                2
-              )}
-        </span>
-      </S.YourVotingPower>
-      <S.TotalVotingPower>
-        <span>
-          {yourVotingPowerInProposal === undefined
-            ? 'total voting power'
-            : 'actual voting power'}
+          </span>
+          <span>
+            {BNtoDecimal(yourVotingPowerInProposal.div(Big(10).pow(18)), 18, 2)}
+          </span>
+        </S.YourVotingPower>
+      )}
 
-          {yourVotingPowerInProposal === undefined ? (
+      {totalVotingPower && (
+        <S.TotalVotingPower>
+          <span>
+            total voting power
             <Tippy content="This is the total voting power across all participants of the Kassandra Decentralized Autonomous Organization in this blockchain.">
               <S.Tooltip tabIndex={0}>
-                <Image src={tooltip} alt="Explanation" width={14} height={14} />
+                <Image
+                  src={tooltip}
+                  alt="Tooltip about voting power of all participants of Kassandra"
+                  width={14}
+                  height={14}
+                />
               </S.Tooltip>
             </Tippy>
-          ) : (
+          </span>
+          <span>{BNtoDecimal(totalVotingPower, 0, 2)}</span>
+        </S.TotalVotingPower>
+      )}
+
+      {yourVotingPowerInProposal && currentVotingPower && (
+        <S.TotalVotingPower>
+          <span>
+            actual voting power
             <Tippy content="This is your actual voting power, it may differ from the snapshot voting power if you earned or lost voting power since the proposal was created.">
               <S.Tooltip tabIndex={0}>
-                <Image src={tooltip} alt="Explanation" width={14} height={14} />
+                <Image
+                  src={tooltip}
+                  alt="tooltip about your current voting power"
```

Comment:
The image is not a tooltip, the alt should describe what is on the image. Since it doesn't matter for the user that is using screenreader, you can add a aria-hidden="true" here. No aria is better than bad aria.

Reply:
What you can do is add a aria-label on the `S.Tooltip` and describe what this button does, instead